calls.go

```diff
@@ -138,15 +145,27 @@ func (call *callsCmd) list(ctx *cli.Context) error {
 		params.PerPage = &per_page
 	}
 
-	resp, err := call.client.Call.GetAppsAppCalls(&params)
-	if err != nil {
-		switch e := err.(type) {
-		case *apicall.GetAppsAppCallsNotFound:
-			return errors.New(e.Payload.Error.Message)
-		default:
-			return err
+	var resCalls []*models.Call
+	for {
```

Comment:
we need to get rid of this too to only return 100 unless they specify the last hour (instead of defaulting to it)

Reply:
I see this a bit differently. `--per-page` is useless that's why it does make sense to replace one with `--n` which is defaulted to 100 objects, so the API will return its default number of calls.
If `--from-time` not specified CLI will return 100 calls from the last hour.

Reply:
sorry, I should have commented on the default last hour line.

Reply:
i see, fixed.